### ISSUE TITLE
Search icon svg has typos that break its a11y

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1571]
```
<svg width="24" height="24" viewBox="0 0 24 24" role="image" aria-labelled-by="bl-icon-search-5549c8aa03693546" class="" data-la-highlighted-by="3cad0e68-7421-1141-150a-a6c5c3dbb81b-toggle">
  <path fill="none" d="M0 0h24v24H0V0z"></path>
  <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"></path>
  <title id="bl-icon-search-5549c8aa03693546">search icon</title>
</svg>
```

- [x] `role="image" ` should be `role="img" `
- [x] `aria-labelled-by`  should be `aria-labelledby`

**Comment by mejackreed** [comment]
I think this needs to happen in Blacklight
